### DISCUSSION TITLE
How to clear the view ?  view.setClearColor() function is missing?

### DISCUSSION THREAD

**Discussion by techGuyAj** [google/filament#4677] [Q&A] [answered]
Hi.
I am upgrading the filament in my Android from the 2019 version to 1.12.4. 
In my current code, I am clearing the screen by using 
`view.setClearColor(1.0f,0.0f,0.0f,0.0f)`

In the latest version, I couldn't find it.

how to set the clear color in the latest version??

Thank you.

**Comment by romainguy** [comment, marked as answer]
You are looking for `Renderer.setClearOptions()`.

**Comment by techGuyAj** [comment]
Got it .
Thank you @romainguy
